Makefile

```diff
@@ -146,9 +120,17 @@ credentials/backup-secret.json:
 	@mkdir -p credentials
 	@echo '$(BACKUP_SECRET)' > credentials/backup-secret.json
 
-ostree-backup: credentials/backup-secret.json ## Backup model VM into ostree container		make ostree-backup SEED_IMAGE=quay.io/whatever/ostmagic:seed
-	scp $(SSH_FLAGS) ostree-backup.sh credentials/backup-secret.json core@$(MODEL_VM_NAME):/tmp
-	ssh $(SSH_FLAGS) core@$(MODEL_VM_NAME) sudo /tmp/ostree-backup.sh $(SEED_IMAGE)
+.PHONY: ibu-imager
+ibu-imager: credentials/backup-secret.json ## Create seed image using ibu-imager		make ibu-imager SEED_IMAGE=quay.io/whatever/ostmagic:seed
+	scp $(SSH_FLAGS) credentials/backup-secret.json core@$(MODEL_VM_NAME):/tmp
+	ssh $(SSH_FLAGS) core@$(MODEL_VM_NAME) sudo podman run --privileged --rm --pid=host --net=host \
+		-v /var:/var \
+		-v /var/run:/var/run \
+		-v /etc:/etc \
+		-v /run/systemd/journal/socket:/run/systemd/journal/socket \
+		-v /tmp/backup-secret.json:/tmp/backup-secret.json \
+		$(IBU_IMAGER) \
```

Comment:
this needs to be updated as well to something similar to below, pls take a look at [HACKING.md](https://github.com/openshift-kni/lifecycle-agent/blob/main/HACKING.md#running-the-ibu-imager) guide

```
podman run --privileged --pid=host --rm --net=host \
    -v /etc:/etc \
    -v /var:/var \
    -v /var/run:/var/run \
    -v /run/systemd/journal/socket:/run/systemd/journal/socket \
    -v ${AUTHFILE}:${AUTHFILE} \
    --entrypoint /ibu-imager ${LCA_IMAGE} create --authfile ${AUTHFILE} \
                                                 --image ${IMG_REPO}
```